packages/dotcom-build-images/src/index.ts

```diff
@@ -0,0 +1,41 @@
+const path = require('path')
+const glob = require('glob')
+const MultiEntryPlugin = require('webpack/lib/MultiEntryPlugin')
+
+export class PageKitImagesPlugin {
+  basePath: string
+
+  constructor(directory = './client') {
```

Comment:
other plugins use an options object instead, which i think is more readable for the consumer, and it'd be easier to add other options in the future